generate_outputs.py

```diff
@@ -78,6 +77,14 @@
     data = TranslateRapidProKeys.translate_rapid_pro_keys(user, data, pipeline_configuration)
 
     if pipeline_configuration.move_ws_messages:
```

Comment:
We should probably move filter_messages() from auto coding stage to a standalone stage before ws correction.

Reply:
Discussed offline.